visidata/_input.py

```diff
@@ -623,6 +625,7 @@ def editCell(self, vcolidx=None, rowidx=None, value=None, **kwargs):
 
     # update local bindings with kwargs.bindings instead of the inverse, to preserve kwargs.bindings for caller
     bindings.update(kwargs.get('bindings', {}))
+    bindings.update({k: acceptThenFunc(*v) for k, v in options.edit_accept_keybindings.items()})
```

Comment:
I like the idea of allowing people to customize their keybindings for edit/input, but the 'always accept' gives me pause every time I come back to it.  I think it'd be better if we had user-set bindings apart from options, and required them to manually specify something like `acceptThenFunc`.  Like:

```
vd.edit_bindings = {
    'Ctrl+X': acceptThenFunc('syscopy-cell', 'delete-cell')
}
```

but that API is pretty terrible too.

Reply:
Thanks again for starting a discussion on this, @ajkerrigan.  I think this isn't quite the way we should go with it, and the input code has been refactored recently anyway.   So we're going to close this for now and find another way (eventually) for people to add custom keybindings in their .visidatarc to the input widget.